src/simulation/simulation.jl

```diff
@@ -840,8 +841,25 @@ function _update_simulation_state!(sim::Simulation, model::DecisionModel)
     simulation_time = get_current_time(sim)
     state = get_simulation_state(sim)
     model_params = get_decision_model_params(store, model_name)
-    for field in fieldnames(DatasetContainer)
+    #Order matters; update parameters first (AvailableStatusChangeCountdownParameter must be updated first if it exists)
+    for field in [:parameters, :duals, :aux_variables, :variables, :expressions]
+        #TODO - fix hardcode for ThermalStandard
```

Comment:
Need to generalize to checking for `AvailableStatusChangeCountdownParameter` for any component